Open Judge System/Web/OJS.Web/Web.config

```diff
@@ -24,6 +24,7 @@
     <add key="webpages:Enabled" value="false" />
     <add key="PreserveLoginUrl" value="true" />
     <add key="ApiKey" value="1234" />
+	<add key="ApiKeySecondary" value="" />
```

Comment:
This might be a little confusing. The `ApiKey` is used from Judge to authenticate itself to other apps (suls), but the `ApiKeySecondary` is used for other apps to authenticate to Judge. In my mind a renaming should be made, because now it seems both are used for the same purpose. Maybe `ApiKey` => `SulsApiKey`, `ApiKeySecondary` => `ApiKey`?